apps/omg_watcher_info/test/omg_watcher_info/api/block_test.exs

```diff
@@ -0,0 +1,56 @@
+# Copyright 2019 OmiseGO Pte Ltd
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+defmodule OMG.WatcherInfo.API.BlockTest do
+  use ExUnitFixtures
+  use ExUnit.Case, async: false
+  use OMG.WatcherInfo.Fixtures
+
+  import OMG.WatcherInfo.Factory
+
+  alias OMG.Utils.Paginator
+  alias OMG.WatcherInfo.API
+  alias OMG.WatcherInfo.DB
+
+  describe "get_blocks/1" do
+    @tag fixtures: [:phoenix_ecto_sandbox]
+    test "returns a paginator with a list of blocks" do
+      _ = insert(:block, blknum: 1000, hash: "0x1000", eth_height: 1, timestamp: 100)
+      _ = insert(:block, blknum: 2000, hash: "0x2000", eth_height: 2, timestamp: 200)
+      _ = insert(:block, blknum: 3000, hash: "0x3000", eth_height: 3, timestamp: 300)
+
+      constraints = []
+      results = API.Block.get_blocks(constraints)
+
+      assert %Paginator{} = results
+      assert length(results.data) == 3
+      assert Enum.all?(results.data, fn block -> %DB.Block{} = block end)
+    end
+
+    @tag fixtures: [:phoenix_ecto_sandbox]
```

Comment:
can we step away from fixtures overall and use the common phoenix-ecto approach?

Reply:
I tried. We have to do it all in one go. I'm already on it but judging from the changes I need so far in https://github.com/omisego/elixir-omg/commit/b6a230ff85112f74b4ded328e606b3c52fee4e7b (only half way through), it better goes in a separate PR.

`:phoenix_ecto_sandbox` is the minimum fixture required and easily replaceable.

Reply:
cool! in this comment I was just referring to :phoenix_ecto_sandbox as it seems something that could really stand in the setup block of a test!

Reply:
Yeah the problem is that (many) other places still call `:phoenix_ecto_sandbox` and their start behavior (starts/stops `DB.Repo` and `Web.Endpoint` per test) conflicts with the common approach (starts once for all). So I had to withdrew.

But yeah, anyhow, coming in another PR.